### PR TITLE
[UXE-2342] feat: pending action reappears

### PR DESCRIPTION
**Pull request by pauloSF0**
## Pull Request
[UXE-2342]

### What is the new behavior introduced by this PR?
 feat: pending action reappears

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/110847590/da546265-8352-476c-a1e3-1c62a9a29b99

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-2342]: https://aziontech.atlassian.net/browse/UXE-2342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/services/real-time-purge/create-real-time-purge-service.js [hunk 1]
```diff
@@ -5,25 +5,42 @@ import { makeRealTimePurgeBaseUrl } from './make-real-time-purge-service'
 export const createRealTimePurgeService = async (payload) => {
   const bodyRequest = adapt(payload)
 
+  const MAP_TYPE = {
+    CacheKey: 'cachekey',
+    Wildcard: 'wildcard',
+    URL: 'url'
+  }
+
+  //when the action is “repurge” I need to get the value of MAP_TYPE
```
Comment (matheus-sunderhus) [inline src/services/real-time-purge/create-real-time-purge-service.js:14]: remover comentário

File: src/services/real-time-purge/create-real-time-purge-service.js [hunk 2]
```diff
@@ -5,25 +5,42 @@ import { makeRealTimePurgeBaseUrl } from './make-real-time-purge-service'
...
+}
+
 const adapt = (payload) => {
   let argumentsPurge = payload.argumentsPurge
   const isArray = Array.isArray(payload.argumentsPurge)
   if (!isArray) {
     argumentsPurge = payload.argumentsPurge.trim().split('\n')
   }
 
+  //when the action is “repurge” I need to get the value of MAP_LAYER
```
Comment (matheus-sunderhus) [inline src/services/real-time-purge/create-real-time-purge-service.js:38]: neste ponto também.

File: src/services/real-time-purge/create-real-time-purge-service.js [hunk 3]
```diff
@@ -5,25 +5,42 @@ import { makeRealTimePurgeBaseUrl } from './make-real-time-purge-service'
...
-    url: `${makeRealTimePurgeBaseUrl()}/${payload.purgeType}`,
+    url: `${makeRealTimePurgeBaseUrl()}/${purgeType}`,
     method: 'POST',
     body: bodyRequest
   })
 
   return parseHttpResponse(httpResponse)
 }
 
+const MAP_LAYER = {
```
Comment (matheus-sunderhus) [inline src/services/real-time-purge/create-real-time-purge-service.js:26]: Usar convenção de MAPLAYER ou mapLayer para declaração da variável ou constante.

File: src/services/real-time-purge/list-real-time-purge-service.js [hunk 4]
```diff
@@ -36,16 +36,27 @@ export const listRealTimePurgeService = async (
   return parseHttpResponse(httpResponse)
 }
 
+const MAP_LAYER = {
```
Comment (matheus-sunderhus) [inline src/services/real-time-purge/list-real-time-purge-service.js:39]: ajuste aqui também para a convenção no projeto de nomenclatura de variáveis.

**Review by matheus-sunderhus — COMMENTED** [review]
minor changes required

**Review by aloisio-m-bastian — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by matheus-sunderhus — APPROVED** [review]
LGTM

**Pull request merged**